src/Resources/public/javascript/autocomplete.js

```diff
@@ -14,55 +14,37 @@ var ZichtQuicklistAutocomplete = (function($) {
         }
 
         $text
-            .focus(function(){$text.select();})
+            .focus(function () {
+                $text.select();
+            })
             .autocomplete({
                 appendTo: ".zicht-ui-autocompletecontainer_" + $text.attr('id'),
-                source: function(req, resp) {
+                source: function (req, resp) {
                     params.pattern = req.term;
                     $.get(
                         service_url,
                         params,
                         resp
                     );
                 },
-                select: function(e, ui) {
+                select: function (e, ui) {
                     $hidden.val(ui.item.id);
                     $text.val(ui.item.label);
                     callback(ui.item);
+                },
+                change: function(e, ui) {
+                    var value = $(e.target).val();
+                    if (value && new RegExp('^(http|https|mailto):', 'i').test(new String(value))) {
+                        $hidden.val(value);
+                    }
```

Comment:
@7ochem Dit is een vrij makkelijk te verwerken aanpassing die alles wat na een `[change](https://api.jqueryui.com/autocomplete/#event-change)` binnenkomt alsnog verwerkt als het niet in de `select` verwerkt is. Alternatief zou bijv. zijn om hier niet op http(s) e.d. te checken maar te kijken of het niet met een `/` begint

Reply:
Dit is nu overigens alleen wenselijk voor het gebruik van de UrlType icm AutocompleteType...evt zou dit gedrag met een FormOption te beinvloeden kunnen zijn, die we dan weer doorgeven via een `attr`, bijv. een `data-allow-manual-input`

Reply:
Ja dat was mijn idee. Een form optie waarmee altijd de waarde naar het hidden veld wordt doorgezet, of het nu voorkomt in de autosuggest of niet.

"autosuggest" in de pure zin des woords geeft suggesties die je **kunt** gebruiken, maar hoeft niet. Wat het nu is, is een select waar je in kun typen en je een waarde uit de suggesties **moet** kiezen. Dus op zich geen gekke feature 😄 

Andere optie zou kunnen zijn een form optie waarmee je direct in het "originele" veld typt i.p.v. een vervangend veld. Dan wordt de waarde ook gebruikt, zonder dat je iets **moet** kiezen. Dat komt dichter in de buurt van de functionaliteit zoals in de Url bundle werd gebruikt. Alleen dit vereist wel veel meer refactoring.